src/main/schemas/audioconfigs.schema.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "http://isdcf.com/ns/json-schemas/registries/audioconfigs/1.0.0-beta.1",
+  "$comment": "Copyright, ISDCF <info@isdcf.com>",
+  "title": "Schema for the Audio Configurations Registry of the ISDCF",
+  "definitions": {
+    "dcncCode" : {
+        "type": "string",
+        "pattern": "^\\[?[A-Z0-9]\\]?"
+    },
+    "cplMetadata": {
+      "type": "object",
+      "properties": {
+          "MainSoundConfigurationTag": {
+              "type": "string"
+          },
+          "MCATagSymbol": {
+              "type": "string"
+          },
+          "extension": {
+            "$ref": "#/definitions/extension"
+          },
+          "definingDoc": {
+              "type": "array"
```

Comment:
```suggestion
              "type": "array",
              "items": {
                "type": "number"
              }
```

Reply:
These values aren't only numbers, the contain values such as:
- "SMPTE ST 428-12"
- "https://www.d-box.com/wp-content/uploads/2020/09/124-915-0005.pdf"

Reply:
I meant `"type": "string"`

Apologies.

Reply:
All good, updated in 23dea0a